source/beamline/BioXAS/BioXASMirrorYawControl.cpp

```diff
@@ -0,0 +1,103 @@
+#include "BioXASMirrorYawControl.h"
+#include "actions3/AMActionSupport.h"
+#include "actions3/AMListAction3.h"
+
+BioXASMirrorYawControl::BioXASMirrorYawControl(const QString &name, const QString &units, QObject *parent, const QString &description) :
+	BioXASMirrorControl(name, units, parent, description)
+{
+	// Initialize inherited variables.
+
+	setContextKnownDescription("Yaw");
+
+	// Current settings.
+
+	updateStates();
+}
+
+BioXASMirrorYawControl::~BioXASMirrorYawControl()
+{
+
+}
+
+bool BioXASMirrorYawControl::canMeasure() const
+{
+	bool result = false;
+
+	if (isConnected()) {
+		result = yaw_->canMeasure();
+	}
+
+	return result;
+}
+
+bool BioXASMirrorYawControl::canMove() const
+{
+	bool result = false;
+
+	if (isConnected()) {
+		result = yaw_->canMove();
```

Comment:
Do you need to check if the ```stripeSelect_``` can move/measure/stop as well since it's inside your move action?  It seems like other implementations check the other associated controls.